src/store/modules/user.ts

```diff
@@ -1,16 +1,31 @@
 import {User} from '@/types';
+import {container} from '@/provider/container';
+import {UserApi} from '@/api';
+import {TYPES} from '@/provider/types';
 
-export const userModule  = {
+
+export const userModule = {
+  namespaced: true,
   state: {
-    user: {} as User,
+    self: {} as User,
   },
   mutations: {
-
+    syncSelf(state, self) {
+      state.self = self;
+    },
   },
   actions: {
-
+    async syncSelf({commit}) {
```

Comment:
i wanna get feedback what action we need.
syncSelf is just example that i can think,

can u guide what action we need?